packages/fx-core/src/plugins/solution/fx-solution/question.ts

```diff
@@ -9,14 +9,19 @@ import {
   TextInputQuestion,
   Void,
 } from "@microsoft/teamsfx-api";
+import { isBotNotificationEnabled } from "../../../common/featureFlags";
 import { getLocalizedString } from "../../../common/localizeUtils";
 
 export const TabOptionItem: OptionItem = {
   id: "Tab",
-  label: "Tab",
+  label: isBotNotificationEnabled() ? "SSO enabled Launch page" : "Tab",
   cliName: "tab",
```

Comment:
Does this apply to cli? Or the wording between vscode and cli will be different.

Reply:
This PR only changes VSC, CLI UX is not finalized yet.